src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -147,7 +135,7 @@ internal unsafe void Register(EventSource eventSource)
             status = EventRegister(eventSource, m_etwCallback);
             if (status != 0)
             {
-#if TARGET_WINDOWS && !ES_BUILD_STANDALONE
+#if TARGET_WINDOWS
                 throw new ArgumentException(Interop.Kernel32.GetMessage(unchecked((int)status)));
```

Comment:
Can this use the Marshal.GetPInvokeErrorMessage and not require ifdef'ing?

Reply:
I can't quickly guarantee it would have identical behavior and I am hoping to keep this change as a mechanical refactoring. GetPInvokeErrorMessage calls Interop.Sys.StrError(error) whereas currently the code invokes Convert.ToString(). I doubt EventRegister on non-Windows platforms was written to return an error code that strerror properly handles.